module/VuFindApi/src/VuFindApi/Controller/SearchApiController.php

```diff
@@ -304,89 +330,161 @@ public function searchAction()
         $request = $this->getRequest()->getQuery()->toArray()
             + $this->getRequest()->getPost()->toArray();
 
+        $isCursorSearch = ($request['resumptionToken'] ?? false);
+        try {
+            if ($isCursorSearch) {
+                $response = $this->doCursorSearch($request);
+            } else {
+                $response = $this->doDefaultSearch($request);
+            }
```

Comment:
This could be simplified:

```suggestion
            $response = $isCursorSearch
                ? $this->doCursorSearch($request)
                : $this->doDefaultSearch($request);
```